model/service_logs/v1/cluster_log_resource.model

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages a specific cluster log.
+resource ClusterLog {
```

Comment:
The first time I read this I thought that `ClusterLog` represents an entire log for a cluster. Then I realized that it only represents one entry. I'd suggest to rename this to `LogEntry` to make that explicit, and to explain it in the documentation.